src/app/common/services/mqtt/mqtt.manager.service.ts

```diff
@@ -78,6 +78,12 @@ export class MqttManagerService {
     this.mqttService.publish(topic, payload).subscribe();
   }
 
+  publishToService(channel: string, svc: string, bn: string, n: string, vs: string) {
+    const topic = this.createTopic(channel) + `services/${svc}`;
```

Comment:
Concat with `${this.createTopic(channel)}/services/${svc}`

Btw, it can be confusing if createTopic() returns `/` as last characters